src/User/resources/views/settings/two-factor-sms.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/*
+* This file is part of the 2amigos/yii2-usuario-app project.
+*
+* (c) 2amigOS! <http://2amigos.us/>
+*
+* For the full copyright and license information, please view
+* the LICENSE file that was distributed with this source code.
+*/
+
+/** @var string $id */
+/** @var string $uri */
+/** @var string $mobilePhoneRegistration */
+?>
+
+<div id="phonenumbersection">
+    <div class="alert alert-info" id="tfmessagephone">
+        <p>
+            <?= Yii::t(
+                'usuario',
+                'Insert the mobile phone number on which to receive text message.'
```

Comment:
*Insert the mobile phone number where you want to receive the SMS*.

The number should be in international format?